### PR TITLE
SONAR-13091 update docs to officially support Docker

### PR DESCRIPTION
**Pull request by MikeBirnstiehl**

### CODE REVIEW

File: sonarqube/content.md [hunk 1]
```diff
@@ -53,187 +53,27 @@ By default, the image will use an embedded H2 database that is not suited for pr
...
+Set up a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
 
-### Use bind-mounted folders or volumes
+### Use bind-mounted volumes
 
-The images contain the SonarQube installation folders at `/opt/sonarqube`. You will need to bind them to your host to override selected files or directories :
+The images contain the SonarQube installation volumes at `/opt/sonarqube`. You will need to bind them to your host to override select files or directories:
 
--	`/opt/sonarqube/conf`: configuration files, such as `sonar.properties`
+-	`/opt/sonarqube/conf`: configuration files, such as `sonar.properties`. **Note:** This volume is only used for SonarQube 7.9.x. For 8.2+, you'll configure your instance using environment variables.
```
Comment (christophelevis) [inline sonarqube/content.md]:
```suggestion
-	`/opt/sonarqube/conf` **7.9.x only**: configuration files, such as `sonar.properties`. This volume is only used for SonarQube 7.9.x. For 8.2+, you'll configure your instance using environment variables. 
```

File: sonarqube/content.md [hunk 2]
```diff
@@ -53,187 +53,27 @@ By default, the image will use an embedded H2 database that is not suited for pr
 
 > Warning: Only a single instance of SonarQube can connect to a database schema. If you're using a Docker Swarm or Kubernetes, make sure that multiple SonarQube instances are never running on the same database schema simultaneously. This will cause SonarQube to behave unpredictably and data will be corrupted. There is no safeguard until [SONAR-10362](https://jira.sonarsource.com/browse/SONAR-10362).
 
-Setup a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
+Set up a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
 
-### Use bind-mounted folders or volumes
+### Use bind-mounted volumes
```
Comment (christophelevis) [inline sonarqube/content.md]: In the same way as for the Database, I think it can be useful to explain that only single instance of SonarQube should use a set of volumes.

**Review by christophelevis — COMMENTED** [review]
About Get Started in Two Minutes, I tend to think it makes sense to keep it here.
But we now have an inconsistent state:  Installation and Upgrade doc is in SQ docs whereas, for Get Started, we redirect to Docker docs.

File: sonarqube/content.md [hunk 3]
```diff
@@ -21,219 +21,37 @@ ulimit -n 65536
...
-	sonar.jdbc.username=sonarqube sonar.jdbc.password=mypassword
-	sonar.jdbc.url=jdbc:postgresql://localhost/sonarqube
-	```
+## First Installation
 
-4.	Drivers for the supported databases (except Oracle) are already provided. Do not replace the provided drivers; they are the only ones supported. For Oracle, copy the JDBC driver into `$SONARQUBE_HOME/extensions/jdbc-driver/oracle`.
+For installation instructions, see Installing the Server from the Docker Image on the [Install the Server](https://docs.sonarqube.org/latest/setup/install-server/) page.
 
-5.	Run the image:
+##Upgrading
```
Comment (christophelevis) [inline sonarqube/content.md]:
```suggestion
## Upgrading 
```

File: sonarqube/content.md [hunk 4]
```diff
@@ -21,219 +21,37 @@ ulimit -n 65536
 ulimit -u 4096
 ```
```
Comment (christophelevis) [inline sonarqube/content.md:23]: Should we keep the "Get Started in Two Minutes Guide" section with a small explanation and link to SonarQube doc as for Installation and Upgrade?

Ex:


```suggestion
## Get Started in Two Minutes Guide

To know how to quickly run a demo instance, see Using Docker on the [Get Started in Two Minutes Guide](https://docs.sonarqube.org/latest/setup/get-started-2-minutes/) page. When you are ready to move to a more sustainable setup, take some time to read the **Configuration** section below.
```

File: sonarqube/content.md [hunk 5]
```diff
@@ -21,219 +21,37 @@ ulimit -n 65536
...
+> **Warning:** Only a single instance of SonarQube can connect to a database schema. If you're using a Docker Swarm or Kubernetes, make sure that multiple SonarQube instances are never running on the same database schema simultaneously. This will cause SonarQube to behave unpredictably and data will be corrupted. There is no safeguard until [SONAR-10362](https://jira.sonarsource.com/browse/SONAR-10362).
 
-Setup a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
+Set up a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
 
-### Use bind-mounted folders or volumes
+### Use bind-mounted volumes
 
-The images contain the SonarQube installation folders at `/opt/sonarqube`. You will need to bind them to your host to override selected files or directories :
+The images contain the SonarQube installation volumes at `/opt/sonarqube`. You will need to bind them to your host to override select files or directories:
```
Comment (christophelevis) [inline sonarqube/content.md]: Is this correct?
> to override select files or directories

File: sonarqube/content.md [hunk 6]
```diff
@@ -21,219 +21,37 @@ ulimit -n 65536
...
 
--	`/opt/sonarqube/conf`: configuration files, such as `sonar.properties`
+-	`/opt/sonarqube/conf`: **Version 7.9.x only:** configuration files, such as `sonar.properties`.
```
Comment (christophelevis) [inline sonarqube/content.md]:
```suggestion
-	`/opt/sonarqube/conf`: **for version 7.9.x only** - configuration files, such as `sonar.properties`.
```

File: sonarqube/content.md [hunk 7]
```diff
@@ -21,219 +21,37 @@ ulimit -n 65536
...
-    -v $SONARQUBE_HOME/logs:/opt/sonarqube/logs \
-    -v $SONARQUBE_HOME/data:/opt/sonarqube/data \
-    %%IMAGE%%
-```
-
-Use of the environment variables `SONARQUBE_JDBC_USERNAME`, `SONARQUBE_JDBC_PASSWORD`, and `SONARQUBE_JDBC_URL` is deprecated and will stop working in future releases.
-
-More recipes can be found [here](https://github.com/SonarSource/docker-sonarqube/blob/master/recipes.md).
-
 ### Customized image
```
Comment (christophelevis) [inline sonarqube/content.md:59]: I wonder if we should still recommend this since we recommend to use env variables. wdyt @michal-duda-sonarsource?

If we keep it, I think it would make sense to update it:
> FROM sonarqube:8.2-community

or even, if it's correct:
> FROM sonarqube:latest

File: sonarqube/content.md [hunk 8]
```diff
@@ -21,219 +21,37 @@ ulimit -n 65536
...
-	    -v $SONARQUBE_HOME/extensions:/opt/sonarqube/extensions \
-	    -v $SONARQUBE_HOME/logs:/opt/sonarqube/logs \
-	    -v $SONARQUBE_HOME/data:/opt/sonarqube/data \
-	    %%IMAGE%%:community-beta
-	```
-
-7.	Browse to `http://yourSonarQubeServerURL/setup` and follow the setup instructions.
-
-8.	Reanalyze your projects to get fresh data.
+For upgrade instructions, see Upgrading from the Docker Image on the [Upgrade the Server] page.
```
Comment (christophelevis) [inline sonarqube/content.md]:
```suggestion
For upgrade instructions, see Upgrading from the Docker Image on the [Upgrade the Server](https://docs.sonarqube.org/latest/setup/upgrading/) page.
```

File: sonarqube/content.md (hunk 5 above)
Comment (MikeBirnstiehl) [reply sonarqube/content.md]: @michal-duda-sonarsource i'm not sure if this language should be removed

Comment (michal-duda-sonarsource) [reply sonarqube/content.md]: We don't want to recommend bind-mounts for new images. In fact I would remove every occurrence of `bind-mounted` to not mislead people about our intention (especially from the name of the whole section). We had a choice between `bind-mounts` vs `volumes` and we went with `volumes` and that's the term we should use to be clear and consistent with docker docs.

As for this sentence, personally I don't understand it. I would rephrase it to: `We recommend creating volumes to store SonarQube data that you want to keep when you update to newer SonarQube version or switch to a higher SonarQube edition. The volumes should be used for the following directories in the container: ...`

File: sonarqube/content.md (hunk 7 above)
Comment (michal-duda-sonarsource) [reply sonarqube/content.md:59]: There is no `8.2-community`, it's just `8.2`, `8.2-de`,`8.2-ee`. Makes sense to update it to `FROM sonarqube:8.2` I think.

File: sonarqube/content.md [hunk 9]
```diff
@@ -23,217 +23,39 @@ ulimit -u 4096
...
 ## Advanced configuration
 
-#### Use parameters via Docker environment variables
-
-The database can be configured with the following SonarQube properties used as environment variables: `sonar.jdbc.username`, `sonar.jdbc.password` and `sonar.jdbc.url`.
-
-```console
-$ docker run -d --name sonarqube \
-    -p 9000:9000 \
-    -e sonar.jdbc.username=sonar \
```
Comment (michal-duda-sonarsource) [inline sonarqube/content.md:223]: We should not recommend `sonar.jdbc....` environment variables ever TBH, it was always weird. It's even an invalid format for env variables- you can't create an env var with dots under linux or macos (`export sonar.jdbc.username="abc"` will throw an error). My opinion is we should recommend `SONARQUBE_JDBC_USERNAME` style env vars for old images and `SONAR_JDBC_USERNAME` style for new images.
